app/shared/video-player/common/ui/VideoControllerState.kt

```diff
@@ -15,25 +15,30 @@ import androidx.compose.runtime.snapshots.SnapshotStateList
  */
 @Composable
 fun rememberVideoControllerState(
-    initialVisible: Boolean = false
+    initialVisible: VisibleState = VisibleState.INVISIBLE
 ): VideoControllerState {
     return remember {
         VideoControllerState(initialVisible)
     }
 }
 
+enum class VisibleState(var value: Boolean) {
```

Comment:
```suggestion
enum class ControllerVisibility(val topBar: Boolean, val bottomBar: Boolean, val floatingBottomEnd: Boolean, val rhsBar: Boolean)
```
分别表示各个部分的 visibility

Reply:
其实还能更进一步用 `value class` 实现, 把这些值都 pack 到一个 int 里 (用 bit 表示), 不过这点空间我们就没必要节省了